docs/package.json

```
@@ -25,7 +25,7 @@
   "devDependencies": {
     "@babel/core": "^7.0.0",
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.6.0",
+    "@ember/test-helpers": "3.3.0",
```

Any reason not to use:

```suggestion
    "@ember/test-helpers": "^3.3.0",
```

---

Yeah, it is a bit confusing indeed! We have a mix of pinned and caret-range version specifiers. Normally, with the recommended setup of renovate, it would pin all devDependencies, see https://docs.renovatebot.com/dependency-pinning/. I haven't looked at the renovate setup, it seems it is not even running here, although we have config file for it! 🤔 

Created https://github.com/CrowdStrike/ember-aria-utilities/issues/37